codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ConstrainedShapeGeneratorCommon.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.rust.codegen.server.smithy.generators
+
+/**
+ * Functions shared amongst the constrained shape generators, to keep them DRY and consistent.
+ */
+
+fun rustDocsNote(typeName: String) =
+    "this is a constrained type because its corresponding modeled Smithy shape has one or more " +
```

Comment:
Capital T?

Reply:
Nope, this message gets rendered as in `Note: <message>` in Rust docs.